Automattic/woocommerce-payments
# Inbox notes: actualize FAQ link for 'Set up refund policy' note

#### Changes proposed in this Pull Request

* Actualize FAQ link for 'Set up refund policy' note

#### Testing instructions

* Open the modified link on WEB browser and ensure the page scrolls down to the proper section of FAQ

-------------------


**Post merge**
